src/pfs/pfs.proto

```diff
@@ -204,6 +204,12 @@ message DeleteRepoRequest {
   bool force = 2;
 }
 
+// DeleteRepoResponse returns the repos that were deleted by a DeleteRepo call.
+message DeleteRepoResponse {
+  // The repos that were deleted, perhaps none.
+  repeated Repo deleted_repos = 1;
```

Comment:
Is there an expectation that this will eventually include system repos? Otherwise, it seems like the single deleted repo would suffice.

Reply:
Hi Bryce, this is to add a response message when "pachctl delete repo XXX" is executed to delete a non-existing (not created) repo saying "No repos deleted". Not sure 

e.x.:
americaswangiri@Iriss-MacBook-Pro bin % ./pachctl delete repo iris_repo1 --project iris1
No repos deleted.

Reply:
Hi Bryce, I'm not sure what you mean this comment: "Also, this is a breaking API change, so we might want to consider delivering this change differently.". I need more context for that.Thanks.

Reply:
DeleteRepo will only ever return 1 Repo.  This should return a boolean or a single Repo.

Let's change this to:
```protobuf
Repo deleted_repo = 1;
```

or 

```protobuf
bool deleted = 1;
```

Reply:
updated as "Repo deleted_repo = 1"